README.md

```diff
@@ -125,4 +144,39 @@ Redirects are located in the `./static/_redirects` file. To add a redirect, simp
 
 # Redirect /old-file.pdf to /new-file.pdf with 301 status code.
 /old-file.pdf /new-file.pdf
-```
\ No newline at end of file
+
+
+```
+# Contributing Code
+
+If you want to contribute, start working through the icon.community repository, navigate to the Github "issues" tab and start looking through issues. We recommed issues labeled "good first issue". These are issues that we believe are particularly well suited for newcomers. If you decide to start on an issue, leave a comment so that other people know that you're working on it. If you want to help out, but not alone, use the issue comment thread to coordinate.
+
+Please see the [ICON Foundation Development Guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical-development/development-guidelines.md)
+for information regarding our development standards and practices.
+
+# Guidelines for contributing
+
+The following is a set of guidelines for contributing to icon.community.
+
+These guidelines are subject to change. Feel free to propose changes to this document in a pull request.
+
+## Pull Request Checklist
+
+Before sending your pull requests, make sure you do the following:
+
+-   Read these [contributing guidelines](README.md).
```

Comment:
Link needs to be fixed

Reply:
Replaced the section with information that is also displayed on the proposed icon.community/contribute page